src/components/AddProductDialog.vue

```diff
@@ -9,18 +9,32 @@
                 <v-spacer></v-spacer>
                 <v-toolbar-items>
                     <v-btn prepend-icon="mdi-content-save" @click="save"
-                        >Add Product</v-btn
+                        >Save</v-btn
                     >
                 </v-toolbar-items>
             </v-toolbar>
             <div class="d-flex flex-column pa-4">
-                <v-switch label="Visible" v-model="visible"></v-switch>
+                <v-switch
+                    color="warning"
+                    hide-details
+                    label="Hide the product from visitors"
+                    v-model="invisible"
+                ></v-switch>
                 <v-text-field
                     clearable
                     label="Internal Product Name"
                     v-model="internalName"
                 >
                 </v-text-field>
+                <v-select
+                    chips
+                    label="Categories"
+                    :items="allCategories"
+                    item-title="name"
+                    multiple
+                    return-object
```

Comment:
do you have a reason to use this?
otherwise you could just set the `item-value` to id and do not have to worry about conversion stuff

Reply:
I will look into that! Returning only the id sounds promising to me!

Reply:
I pushed changes regarding this v-select, see [this commit](https://github.com/MiSArch/frontend/pull/12/commits/c608d577bfe81337b21dafc9bafa5094bd84341d).